hw11.py

```diff
@@ -0,0 +1,48 @@
+### decorated function ###
+from datetime import datetime
+def decorator(my_func):
+    def deco_func(*args, **kwargs):
+        print(my_func.__name__)
+        print(datetime.now())
+        return my_func()
+    return deco_func(*args, **kwargs)

```

Comment:
Do not call this function since it will bel called immediately after the decorator is applied. But it should not work like this. Here, by `deco_func` we mean a function that is decorated. And this function should be executed in a place where it is called, but not in a place where it is decorated and written.
Leave it as `return deco_func`